mmv1/products/vertexai/IndexEndpoint.yaml

```diff
@@ -97,3 +101,16 @@ properties:
       [Format](https://cloud.google.com/compute/docs/reference/rest/v1/networks/insert): `projects/{project}/global/networks/{network}`.
       Where `{project}` is a project number, as in `12345`, and `{network}` is network name.
     immutable: true
+  - !ruby/object:Api::Type::NestedObject
+    name: privateServiceConnectConfig
+    immutable: true
+    description: |-
+      Optional. Configuration for private service connect. `network` and `privateServiceConnectConfig` are mutually exclusive.
```

Comment:
If these fields are mutually exclusive, consider adding `conflicts:` to both fields

Reply:
I didn't know about the keyword: `conflicts`. Will put them. Thank you!
https://github.com/GoogleCloudPlatform/magic-modules/blob/2a1027ff7d33201dcf512ce58b5438ac4ec59867/mmv1/api/type.rb#L82-L84

Reply:
I've defined `conflicts` in [4970888](https://github.com/GoogleCloudPlatform/magic-modules/pull/8851/commits/4970888ca3e1487885983b388c960d644d7b68c4).